ci/crossbeam-epoch.sh

```diff
@@ -20,4 +20,9 @@ if [[ "$TRAVIS_RUST_VERSION" == "nightly" ]]; then
         --target x86_64-unknown-linux-gnu \
         --features sanitize,nightly \
         --example sanitize
+
+    # Check for no_std environment.
+    cd .. && cargo run --manifest-path ci/Cargo.toml --bin remove-dev-dependencies */Cargo.toml && cd -
+    cargo check --target thumbv7m-none-eabi --no-default-features
```

Comment:
If build with Linux as a target, it will link to std, so it need to specify a target that does not have std.